docs/GenericsManifesto.md

```diff
@@ -284,10 +284,22 @@ protocol P {
 }
 ```
 
-In the above example `Base` may be any type.  `Derived` may be the same as `Base` or may be _any_ subtype of `Base`.  All subtype relationships supported by Swift should be supported in this context including (but not limited to) classes and subclasses, existentials and conforming concrete types or refining existentials, `T?` and  `T`, `((Base) -> Void)` and `((Derived) -> Void)`, etc.
+In the example above `Base` may be any type. `Derived` may be the same as `Base` or may be _any_ subtype of `Base`.  All subtype relationships supported by Swift should be supported in this context including (but not limited to) classes and subclasses, existentials and conforming concrete types or refining existentials, `T?` and  `T`, `((Base) -> Void)` and `((Derived) -> Void)`, etc.
 
 Generalized supertype constraints would be accepted in all syntactic locations where generic constraints are accepted.
 
+There is also a protocol conformance constraint, which is a different kind of constraint. But these two constraints are often confused with each other because both use the `:` symbol and protocols can be used as types.
+
+Currently it is not possible to express existential supertype constraint. Since `:` symbol is overloaded to represent both supertype and protocol conformance constraints, disambiguating between two requires syntax to disambiguate between protocol and its existential. Assuming that syntax is `any P` for existential:
```

Comment:
I think you should break this out into its own section. This is going to require two features: a spelling for upper bounds on existentials and existentials in positions that previously required concrete types.

Reply:
Note that you actually only need the latter to accomplish this feature today as our subtyping relation generalizes to protocol(-composition)s

Reply:
> a spelling for upper bounds
Since `:` is already overloaded to mean either subtyping or conformance, I think we can continue using it.

> existentials in positions that previously required concrete types
What do you mean by `concrete types`?